coordinator/src/errors/errors.ts

```diff
@@ -58,18 +57,15 @@ class FatalError implements ValidationError {
 
 export class NoEntryInBundleError extends FatalError {
   apiErrorCode: "MISSING_FIELD"
-
-  constructor() {
-    super()
-    this.message = "ResourceType Bundle must contain 'entry' field"
-  }
+  message = "ResourceType Bundle must contain 'entry' field"
 }
 
 export class RequestNotBundleError extends FatalError {
   apiErrorCode = "INCORRECT_RESOURCETYPE"
+  message = "ResourceType must be 'Bundle' on request"
+}
 
-  constructor() {
-    super()
-    this.message = "ResourceType must be 'Bundle' on request"
-  }
+export class MessageTypeError extends FatalError {
+  apiErrorCode = "INVALID_VALUE"
+  message = `MessageType must be one of ${MessageType.PRESCRIPTION} or ${MessageType.CANCELLATION}`
```

Comment:
I think this should read "MessageHeader.eventCoding.code must be one of..." as MessageType is something internal to our codebase